packages/@okta/vuepress-site/docs/guides/sign-in-overview/main/index.md

```diff
@@ -37,7 +37,7 @@ After you have an idea of the above considerations, you can integrate Okta authe
 | **Maintenance** | <span style="width: 50%;display:block">![Low](/img/ratings/low.png)</span> No updates required | <span style="width: 50%;display:block">![Medium](/img/ratings/medium.png)</span> Requires updating widget packages | <span style="width: 50%;display:block">![High](/img/ratings/high.png)</span> Requires updating SDK packages and adapting to changes |
 | **Security** | The most secure option. No XSS attacks, Okta takes care of it all | Responsibility shared with Okta | Security is your responsibility |
 | **Customization** | Managed branding and customization options for domains, emails, sign-in page, and more. | Most substantial customization options | Full customization |
-| **Learn more** | Redirect auth for [web apps](/docs/guides/sign-into-web-app-redirect/), [mobile apps](/docs/guides/sign-into-mobile-app-redirect/), and [single-page apps](/docs/guides/sign-into-spa-redirect/).<br>[Redirect auth use cases](/docs/guides/sampleapp-oie-redirectauth/android/main/)<br>[Customize the hosted widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget) | [Embedded widget fundamentals](/docs/guides/embedded-siw/)<br>[Sample app setup process](/docs/guides/oie-embedded-common-org-setup/android/main/)<br>[Embedded widget use cases](/docs/guides/oie-embedded-widget-use-case-load/)<br>[Customize the embedded widget](/docs/guides/custom-widget/main/#style-the-self-hosted-sign-in-widget) | [Auth JS fundamentals](/docs/guides/auth-js/)<br>[Sample app setup process](/docs/guides/oie-embedded-common-org-setup/android/main/)<br>[Embedded SDK use cases](/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/)<br>[Okta SDKs reference](/code/)|
+| **Learn more** | <ul><li>Redirect auth for <a href="/docs/guides/sign-into-web-app-redirect/">web apps</a>, <a href="/docs/guides/sign-into-mobile-app-redirect/">mobile apps</a>, and <a href="/docs/guides/sign-into-spa-redirect/">single-page apps</a></li><li><a href="/docs/guides/sampleapp-oie-redirectauth/android/main/">Redirect auth use cases</a></li><li><a href="/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget">Customize the hosted widget</a></li></ul> | <ul><li><a href="/docs/guides/embedded-siw/">Embedded widget fundamentals</a></li><li><a href="/docs/guides/oie-embedded-common-org-setup/android/main/">Sample app setup process</a></li><li><a href="/docs/guides/oie-embedded-widget-use-case-load/">Embedded widget use cases</a></li><li><a href="/docs/guides/custom-widget/main/#style-the-self-hosted-sign-in-widget">Customize the embedded widget</a></li></ul> | <ul><li><a href="/docs/guides/auth-js/">Auth JS fundamentals</a></li><li><a href="/docs/guides/oie-embedded-common-org-setup/android/main/">Sample app setup process</a></li><li><a href="/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/">Embedded SDK use cases</a></li><li><a href="/code/">Okta SDKs reference</a></li></ul>|
```

Comment:
use the Markdown convention for links:
`<ul><li>Redirect auth for [web apps](/docs/guides/sign-into-web-app-redirect/), [mobile apps](/docs/guides/sign-into-mobile-app-redirect/), [single-page apps](/docs/guides/sign-into-spa-redirect/).</li><li>[Redirect auth use cases](/docs/guides/sampleapp-oie-redirectauth/android/main/)</li><li>[Customize the hosted widget](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget)</li></ul>`

Reply:
Good call. I updated this in my last commit.